womtool/src/main/scala/cromwell/pipeline/womtool/WomTool.scala

```diff
@@ -0,0 +1,78 @@
+package cromwell.pipeline.womtool
+
+import java.nio.file.{ Files, Paths }
+
+import cats.data.NonEmptyList
+import com.typesafe.config.ConfigFactory
+import common.Checked
+import cromwell.core.path.{ DefaultPath, DefaultPathBuilder, Path }
+import cromwell.languages.LanguageFactory
+import cromwell.languages.util.ImportResolver.ImportResolver
+import languages.cwl.CwlV1_0LanguageFactory
+import languages.wdl.biscayne.WdlBiscayneLanguageFactory
+import languages.wdl.draft2.WdlDraft2LanguageFactory
+import languages.wdl.draft3.WdlDraft3LanguageFactory
+import wom.executable.WomBundle
+import womtool.WomtoolMain.{ BadUsageTermination, SuccessfulTermination, Termination, UnsuccessfulTermination }
+import womtool.inputs.Inputs
+
+import scala.util.Try
+
+class WomTool extends WomToolAPI {
+
+  def validate(
+    content: String,
+    importResolvers: List[ImportResolver]
+  ): Either[NonEmptyList[String], (WomBundle, LanguageFactory)] = {
```

Comment:
I don't think we would ever need `LanguageFactory` as a result of this function, I suggest we do without it, `WomBundle` would be enough:
```scala
Either[NonEmptyList[String], WomBundle]
```

Reply:
Done